app/controllers/application_controller.rb

```diff
@@ -11,6 +12,24 @@ def cors_preflight
 
   private
 
+  rescue_from "Exception" do |exception|
```

Comment:
I know it is bad practice to rescue from Exception, but there is no other way to do what I'm trying to do here. I think as long as we log and / or raise the exception again we're ok?